stringer.go

```diff
@@ -529,14 +567,14 @@ func (g *Generator) createIndexAndNameDecl(run []Value, typeName string, suffix
 	nameConst := fmt.Sprintf("_%sName%s = %q", typeName, suffix, b.String())
 	nameLen := b.Len()
 	b.Reset()
-	fmt.Fprintf(b, "_%sIndex%s = [...]uint%d{0, ", typeName, suffix, usize(nameLen))
+	_, _ = fmt.Fprintf(b, "_%sIndex%s = [...]uint%d{0, ", typeName, suffix, usize(nameLen))
 	for i, v := range indexes {
 		if i > 0 {
-			fmt.Fprintf(b, ", ")
+			_, _ = fmt.Fprintf(b, ", ")
 		}
-		fmt.Fprintf(b, "%d", v)
+		_, _ = fmt.Fprintf(b, "%d", v)
 	}
-	fmt.Fprintf(b, "}")
+	_, _ = fmt.Fprintf(b, "}")
```

Comment:
Definitely not worth this change. Could you please remove the changes you did to include `_, _ = ` ?